From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Mon, 12 Mar 2018 15:28:14 -0700
Subject: [PATCH] Set m4 macro directory

Fix 'autoreconf' so that the m4 directory is properly read for autoconf
to produce a valid configure script.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index dc0c39e..9db73a5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -33,6 +33,7 @@ m4_define([R_VERSION],
 AC_INIT([R],[R_VERSION],[https://bugs.r-project.org],[R],[https://www.r-project.org])
 AC_CONFIG_SRCDIR([src/include/Defn.h])
 AC_CONFIG_AUX_DIR([tools])
+AC_CONFIG_MACRO_DIRS([m4])
 
 ### * Information on the package.
 
